soc/nxp/lpc/lpc11u6x/Kconfig.defconfig

```diff
@@ -0,0 +1,17 @@
+# Kinetis K2x series configuration options
```

Comment:
Comment here seems wrong.... If we copy pasted this file, should check copyrights as well